scripts/export_requirements.py

```diff
@@ -149,16 +230,23 @@ def main(req_path: os.PathLike, should_validate_hash: bool = True) -> typing.Opt
         dependency_lines[k] = line
 
     req_txt += "\n".join(sorted(k + v.rstrip() for k, v in dependency_lines.items())) + "\n"
+
+    if export_doc_requirements:
+        code = _export_doc_requirements(lockfile, DOC_REQUIREMENTS)
+    else:
+        code = 0
+
     if req_path.exists():
         with open(req_path, "r") as f:
             if req_txt == f.read():
                 # nothing to edit
-                return 0
+                return 0 or code
```

Comment:
`0` is falsy so wouldn't it always return `code`?

Reply:
yes, but I left it returning `0 or` here so it would be clear that this part returns zero, so it is deferred to code

Reply:
> would be clear that this part returns zero


Huh? `0` is falsy so it would never be returned (it would always return `code`) unless `code` itself is `0`.

Reply:
Updated and made a comment about it